deploy/bicep/modules/capps-env.bicep

```diff
@@ -26,13 +29,23 @@ resource appInsights 'Microsoft.Insights/components@2020-02-02' = {
   }
 }
 
-resource containerAppsEnv 'Microsoft.App/managedEnvironments@2022-06-01-preview' = {
+resource containerAppsEnv 'Microsoft.App/managedEnvironments@2022-11-01-preview' = {
   name: containerAppsEnvName
   location: location
-  sku: {
-    name: 'Consumption'
-  }
+  
   properties: {
+    vnetConfiguration: {
+      infrastructureSubnetId: ((!empty(vnetSubnetId)) ? vnetSubnetId : null)
+      internal: ((!empty(vnetSubnetId)) ? true : false)
+    }
+    workloadProfiles: [
+      {
+        minimumCount: 1
+        maximumCount: 10
+        name: workloadProfileName
+        workloadProfileType: workloadProfileType
+      }
+    ]
```

Comment:
To provide a bit more context about what's happening here... setting the `workloadProfile` parameter to an empty array (e.g., `[]`) will keep the ACA environment at the traditional *Consumption* plan. Which is great, but... since we're setting the `workloadProfileName` on the apps... it tries to match to the right profile, and the *Consumption* plan has no matching workload profiles which causes a failure ultimately. This syntax will force the ACA environment to the new *Consumption + Dedicated* plan, but only provides the built-in Consumption workload profile.

Reply:
Fixed in commit #36